packages/api/test/e2e/cennzx.e2e.ts

```diff
@@ -92,7 +90,7 @@ describe('CENNZX RPC calls testing', () => {
         it('Query estimated fee in CENTRAPAY(default fee currency)', async done => {
           const assetBalanceBefore: Balance = await api.query.genericAsset.freeBalance(CENTRAPAY, alice.address);
           const extrinsic = api.tx.genericAsset
-            .transfer(PLUG, bob.address, 10000);
```

Comment:
since now we don't have balance configured for PLUG token on node for endowed accounts.. I have changed this test case to use CENNZ to check fee in default currency